x/config/chooser.go

```diff
@@ -68,62 +68,62 @@ import (
 //
 // Integration
 //
-// To integrate peer lists with your transport, embed this struct into your
+// To integrate peer choosers with your transport, embed this struct into your
 // outbound configuration.
 //
 // 	type myOutboundConfig struct {
-// 		config.PeerList
+// 		config.PeerChooser
 // 		Address string
 // 	}
```

Comment:
Can we use good style here and separate the embedded and named fields with a blank line?